### ISSUE TITLE
Indicator Conversion Issue for property-based timeseries 

### ISSUE THREAD

**Issue by cDanowski** [KomMonitor/kommonitor-importer#16]
Hi,

another issue has occured. The importer was called with a GeoJSON file that contained property-based timeseries (timestamp mapped in property `timestamp`), i.e. each feature was inserted mutliple times to map the time series (10 values +). 

The issue was that the `List<IndicatorPOSTInputTypeIndicatorValues>` within `IndicatorPUTInputType`, produced by the importer, which is then submitted to Data Management API, also contained each feature multiple times. One feature per timestamp as in the original GeoJSON.

The Data Management API expects to have each feature only once, but with an array containing all indicatorValue-timestamp-pairs.

The issue occurred while trying to update an existing indicator. I guess that the issue will also occur when creating a new indicator with property-based time series mapping. I did not test the latter.

It would be very nice if the importer could be changed in order to always send exactly one indicator feature with an array containing all timestamp-indicatorValues pair.

Thanks and regards,
Christian
